README.md

```diff
@@ -31,9 +31,25 @@ Use following Makefile commands:
 # Testing
 To setup accounts passphrase you need to setup an environment variable: `export ACCOUNT_PASSWORD="secret_pass_phrase"`.
 
-To test statusgo, use: `make ci`. Make sure the `gometalinter` package is installed first by running `make lint-install`
-To test statusgo using a giving network by name, use: `make ci networkid=rinkeby`.
-To test statusgo using a giving network by id number, use: `make ci networkid=3`.
+Make sure the dependencies are installed first by running:
+```
+make lint-install
+make mock-install
+```
+To test fully statusgo, use:
+```
+make ci
+```
+
+To test statusgo using a giving network by name, use:
```

Comment:
English is not my first language, but I believe it should be `using a given network`.

Reply:
oops, didn't noticed it, it's fixed now :)